clang/lib/CConv/Constraints.cpp

```diff
@@ -274,16 +275,9 @@ static bool do_solve(ConstraintsGraph &CG,
         if ((doLeastSolution && *Cbound < *Csol) ||
             (!doLeastSolution && *Csol < *Cbound)) {
           ok = false;
-          // Failed. Make a constraint to represent it.
-          std::string str;
-          llvm::raw_string_ostream os(str);
-          os << "Bad solution: "; Csol->print(os);
-          os.flush();
-          Geq *failedConstraint =
-              doLeastSolution ?
-              new Geq(VA, Cbound, str, doLeastSolution) :
-              new Geq(Cbound, VA, str, doLeastSolution);
-          Conflicts.insert(failedConstraint);
+          // Save effected VarAtom in conflict set. This will be constrained to
+          // wild after pointer type solving is finished.
+          Conflicts.insert(VA);
```

Comment:
What about pointers that are transitively affected by this one? Mention that perhaps?